### ISSUE TITLE
Migrate CTN to feature dataset

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#7239]
- Create dataset
- Move CTN segments into dataset
- Update paths to location

**Comment by johnclary** [comment]
Going to do the November segment update on old and new CTN datasets. Todo:
- copy models to work with new CTN
- run updates
- if all goes well, retire the old CTN.

**Comment by amenity** [comment]
@andrewshensky @johnclary - any updates on this?

**Comment by andrewshensky** [comment]
@amenity - yeah, we have migrated the existing CTN data to a new layer in a dataset in the Maint enterprise geodatabase. We have remapped nearly all paths and workflows to use this new CTN layer. We have also added attribute rules and topology rules to improve data quality. I will move this issue to Review/QA since we are still passing the torch from the old to new CTN. January will be the first month where updates will be made exclusively to the new CTN and the existing CTN will exist only as a legacy layer in its current location. Let me know if you have any additional questions.
